cpp/harvest/issn_lookup.py

```diff
@@ -0,0 +1,82 @@
+#!/bin/python3
+#
+#    @brief  The automation script to organise process of checking whether the issn is in k10plus if so then get the data and put it in output file.
+#           The output file format is mrc.
+#
+#    @author Steven Lolong (steven.lolong@uni-tuebingen.de)
+#
+#    Copyright (C) 2023 Library of the University of Tübingen
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import re
+import sys
+import traceback
+import pexpect
+
+
+def ConnectToYAZServer():
+    base_url = "sru.k10plus.de/opac-de-627"
+    marc_charset = "MARC8/UTF8"
+    output_format = "marc21"
+
+    yaz_client = pexpect.spawn("yaz-client")
+    yaz_client.sendline("open " + base_url)
+    yaz_client.sendline("marccharset " + marc_charset)
+    yaz_client.sendline("format " + output_format)
+
+    return yaz_client
+
+
+def Main():
+    if len(sys.argv) != 3:
+        print("[usage]\n\tpython issn_lookup_in_bnb.py input_file output_file")
```

Comment:
Maybe you could add some more information to the usage
- input file should be txt with 1 ISSN per line
- output file will be mrc with all journals found for these ISSNs